src/formatter.js

```diff
@@ -80,6 +80,9 @@ assign(Formatter.prototype, {
     if (typeof value === 'function') {
       return this.outputQuery(this.compileCallback(value), true);
     }
+    if (value instanceof QueryBuilder) {
+      return this.parameterize(value);
+    }
```

Comment:
`raw = this.unwrapRaw(value);` in the next line already runs converting QueryBuilder instance to query...